src/app/app.module.ts

```diff
@@ -37,13 +37,14 @@ import {TemplateEditorComponent} from './dts/template-editor/template-editor.com
     MatButtonModule,
     MatInputModule,
     MatSelectModule,
-    MatTabsModule
+    MatTabsModule,
+    MatDialogModule
   ],
   providers: [
     HttpClient
   ],
   bootstrap: [],
-  entryComponents:[
+  entryComponents: [
```

Comment:
The entry components array is deprecated since Angular 9+ and as a result can be removed. (https://angular.io/guide/entry-components#the-entrycomponents-array - I actually made authored that PR on the Angular website :-) )

Reply:
Removed entryComponents.